guardian/manage-frontend
# Upgrade node riffraff artefact -> artifact

## What does this change?
This PR upgrades us to `@guardian/node-riffraff-artifact`. The library we were using is not being actively maintained. This one should bring some improvements, particularly better logging.

## How to test
Two ways to test:

1. Run `yarn node-riffraff-artifact --dryRun` locally to confirm artifact config is working correctly. See the [library page](https://github.com/guardian/node-riffraff-artifact#how-to-use) for more details on expected behaviour
1. Attempt to deploy on CODE to confirm that the artifact config is working correctly with the CloudFormation and Riff-Raff config.

## How can we measure success?
Our continuous deployment pipelines still works. Which it does.

![image](https://user-images.githubusercontent.com/48949546/122027780-4f702b00-cdc3-11eb-94d6-c59ed041c460.png)
